vignettes/site_only/db_connections.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+title: "Database Connections using healthcare.ai"
+author: "Mike Mastanduno"
+date: "4/1/2018"
+output: html_document
+---
+
+**Database connections**
+
+Let's be honest. You need to get data from a database, do some really awesome 
+stuff with `R`, and then write the results back to a database. `R` can help
+with that. There are lots of different databases, but this document will focus
+on Microsoft SQL Server. R supports most databases, and 
+I'd encourage you to look at their [help pages](https://db.rstudio.com/) 
+for additional resources. Andiamo!
+
+If you simply want a code snippet to copy and paste, scroll to the bottom.
+
+# Reading data
```

Comment:
Top-level section headers go into a table of contents. I'd put Connecting to a Database at the top level, before Reading data

Reply:
Good call.